paddle/fluid/operators/controlflow/tensor_array_read_write_op.cc

```diff
@@ -82,16 +82,23 @@ equation is
 class WriteToArrayInferShape : public framework::InferShapeBase {
  public:
   void operator()(framework::InferShapeContext *context) const override {
-    PADDLE_ENFORCE(context->HasInput("I"), "Must set the subscript index");
-    if (context->IsRuntime()) {
-      PADDLE_ENFORCE_EQ(framework::product(context->GetInputDim("I")), 1,
-                        "The number of element of subscript index must be 1");
-    }
+    PADDLE_ENFORCE_EQ(
+        context->HasInput("I"), true,
+        platform::errors::InvalidArgument(
+            "Write to array operation must set the subscript index."));
+
+    // TODO(wangchaochaohu) control flow Op do not support runtime infer shape
+    // Later we add [ontext->GetInputDim("I")) == 1] check when it's supported
+
     if (!context->HasInput("X")) {
       return;
     }
 
-    PADDLE_ENFORCE(context->HasOutput("Out"), NotHasOutError());
+    PADDLE_ENFORCE_EQ(
+        context->HasOutput("Out"), true,
+        platform::errors::InvalidArgument(
+            "Do array operation (write/read)must set the output Tensor "
```

Comment:
Tiny English issue: 

1. What's "Do array operation"? Did you try to say "Write to array operation"? 
2. ")must" -> ") must".    That is adding a space.

Reply:
Because read and write infer shape is same, so we comment it like this: Read/Write